whipper/common/program.py

```diff
@@ -470,6 +471,70 @@ def getHTOA(self):
         stop = track.getIndex(1).absolute - 1
         return start, stop
 
+    def getCoverArt(self, path, cover_art, release_id):
+        """
+        Get cover art image from Cover Art Archive if user chooses to
+
+        :type  cover_art: bool
+        :param cover_art: whether to fetch cover art or not
+        :type  path: str
+        :param path: where to store the fetched image
+        :type  release_id: str
+        :param release_id: a release id (self.program.metadata.mbid)
+
+        :rtype: str or None
+        """
+        if not cover_art:
+            return
+
+        import requests
+        import whipper
+
+        image_path = os.path.join(path, 'coverart.jpg')
```

Comment:
I would propose to use either `cover.jpg` or `front.jpg` here, as those are the names generally recognised by media players etc., in case the artifact doesn’t get cleaned up. (Or for easier reuse if you add an option to save the image next to the ripped audio files. In this case you may also consider not hardcoding the file extension… and allowing non-JPEG/PNG files…)

Reply:
Picard defaults to saving to `cover.$ext` (with `$ext` being the relevant file extension): https://picard.musicbrainz.org/docs/options/#location

Reply:
> In this case you may also consider not hardcoding the file extension… and allowing non-JPEG/PNG files…)

Yeah, the current code always saves the cover art file with the `jpg` extension (even if it's a `png`) and that's wrong.